### PR TITLE
add MKS_32_V1_4 variant

### PR DESCRIPTION
**Pull request by darkspr1te**
Added MKS TFT 28/32" screen variant
supports bootloader/no bootloader models
uart2 - > aux1 ramps board
uart1 wifi txd/rxd
uart3 wifi tx,rx 
sd card - spi1 
eeprom - spi1
touch spi2 
i2cprom - i2c1 
usb host enabled
pb0/pb1 -> filiment runout/power
pb4/pb5 - > ps ok/unused

### CODE REVIEW

**Comment by darkspr1te** [comment]
I see from the log it's failing at line 21 in main.c , that line is only included when you have MKS_32_V1_4 defined yet the build environment is including it under TFT 35 V3, i'am confused why.

File: TFT/src/User/main.c [hunk 1]
```diff
@@ -18,14 +18,19 @@ void Hardware_GenericInit(void)
   RCC_APB2PeriphClockCmd(RCC_APB2Periph_AFIO , ENABLE);
   GPIO_PinRemapConfig(GPIO_Remap_SWJ_JTAGDisable, ENABLE);
 #endif
-  
+  RCC_APB2PeriphClockCmd(RCC_APB2Periph_AFIO,ENABLE);
```
Comment (guruathwal) [inline TFT/src/User/main.c]: This line is not required.
```suggestion
```

File: TFT/src/User/main.c [hunk 2]
```diff
@@ -18,14 +18,19 @@ void Hardware_GenericInit(void)
   RCC_APB2PeriphClockCmd(RCC_APB2Periph_AFIO , ENABLE);
   GPIO_PinRemapConfig(GPIO_Remap_SWJ_JTAGDisable, ENABLE);
 #endif
-  
+  RCC_APB2PeriphClockCmd(RCC_APB2Periph_AFIO,ENABLE);
+  GPIO_PinRemapConfig(GPIO_Remap_USART2, ENABLE);
```
Comment (guruathwal) [inline TFT/src/User/main.c]: This line is not required.
```suggestion
```

File: platformio.ini [hunk 3]
```diff
@@ -18,7 +18,7 @@
 [platformio]
 src_dir      = TFT
 boards_dir   = buildroot/boards
-default_envs = MKS_32_V1_4_NOBL
+default_envs = MKS_32_V1_4
```
Comment (guruathwal) [inline platformio.ini]: this should be `BIGTREE_TFT35_V3_0' default for the Repo.
```suggestion
default_envs = BIGTREE_TFT35_V3_0
```

**Pull request merged**